src/sentry/snuba/models.py

```diff
@@ -40,6 +40,11 @@ class Type(Enum):
     aggregate = models.TextField()
     time_window = models.IntegerField()
     resolution = models.IntegerField()
+    # TODO: Modify snuba query creation to make use of `start_time` if provided (eg. query all events _after_ provided start_date)
+    start_time = models.DateTimeField(null=True, default=None)
+    release = FlexibleForeignKey(
```

Comment:
If i want a query created to filter on a particular release, do i need to add the FK here?
Or would that just go into the `query` string above?

Reply:
Actually - i suppose the `query` string is set by the user, so best not to modify that 🤔 

will keep this in place